specification/monitor/resource-manager/Microsoft.Insights/stable/2023-12-01/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+        },
+        "operator": {
+          "type": "string",
+          "enum": [
+            "Equals",
+            "GreaterThan",
+            "GreaterThanOrEqual",
+            "LessThan",
+            "LessThanOrEqual"
+          ],
+          "x-ms-enum": {
+            "name": "conditionOperator",
+            "modelAsString": true
+          },
+          "description": "The criteria operator. Relevant and required only for rules of the kind LogAlert."
+        },
+        "threshold": {
+          "type": "number",
+          "format": "double",
+          "description": "the criteria threshold value that activates the alert. Relevant and required only for rules of the kind LogAlert."
+        },
+        "failingPeriods": {
+          "type": "object",
+          "description": "The minimum number of violations required within the selected lookback time window required to raise an alert. Relevant only for rules of the kind LogAlert.",
+          "properties": {
+            "numberOfEvaluationPeriods": {
+              "type": "integer",
+              "format": "int64",
+              "default": 1,
+              "description": "The number of aggregated lookback points. The lookback time window is calculated based on the aggregation granularity (windowSize) and the selected number of aggregated points. Default value is 1"
+            },
+            "minFailingPeriodsToAlert": {
+              "type": "integer",
+              "format": "int64",
+              "default": 1,
+              "description": "The number of violations to trigger an alert. Should be smaller or equal to numberOfEvaluationPeriods. Default value is 1"
+            }
+          }
+        },
+        "metricName": {
+          "type": "string",
+          "description": "The name of the metric to be sent. Relevant and required only for rules of the kind LogToMetric."
+        }
+      }
+    },
+    "Dimension": {
+      "type": "object",
+      "description": "Dimension splitting and filtering definition",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the dimension"
+        },
+        "operator": {
+          "type": "string",
+          "description": "Operator for dimension values",
+          "enum": [
+            "Include",
+            "Exclude"
+          ],
+          "x-ms-enum": {
+            "name": "dimensionOperator",
+            "modelAsString": true
+          }
+        },
+        "values": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of dimension values"
+        }
+      },
+      "required": [
+        "name",
+        "operator",
+        "values"
+      ]
+    },
+    "ErrorContract": {
+      "description": "Describes the format of Error response.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
```

Comment:
Please delete this line -- I think this is what is causing the breaking changes tooling to complain.
```suggestion
```

Reply:
Thank you @mikekistler. I actually just pushed a fix right now, which references the common-types "ErrorContract" directly inside the "schema" field, instead of going through the locally defined wrapper "ErrorContract". I hope this will resolve it.